From 39f7c437f752d93bddf37f1f4be7d4cb0a41cc9a Mon Sep 17 00:00:00 2001
From: cppbitman <chinadaihongjun@gmail.com>
Date: Tue, 23 Oct 2018 05:46:39 -0700
Subject: [PATCH] convert PKGCONFIG_CAIRO_REQUIRES value to string

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ea7f5ff..0c0f156 100644
--- a/meson.build
+++ b/meson.build
@@ -326,7 +326,7 @@ pkgconf.set('libdir', pango_libdir)
 pkgconf.set('includedir', pango_includedir)
 pkgconf.set('VERSION', meson.project_version())
 pkgconf.set('PANGO_API_VERSION', pango_api_version)
-pkgconf.set('PKGCONFIG_CAIRO_REQUIRES', pangocairo_requires)
+pkgconf.set('PKGCONFIG_CAIRO_REQUIRES', ' '.join(pangocairo_requires))
 
 pkgconf_files = [
   [ 'pango.pc' ],
-- 
2.17.1

